modules/ROOT/pages/notifications/gqlstatus-info-codes.adoc

```diff
@@ -0,0 +1,62 @@
+:description: The page contains GQLSTATUS info codes producing results such as SUCCESS, NO DATA, and OMITTED RESULT.
+
+[[gql-status-info-codes]]
+= Primary GQLSTATUS codes
+
+GQL requires several new statuses that are not covered by the current Neo4j notification framework.
+
+.GQL status information codes
+[cols="1,1,1,1", options="header"]
+|===
+|GQLSTATUS
+|Condition
+|Subcondition
+|Description
+
+| 00000
+| Successful completion
+|
+| Successful completion with a regular non-empty result
+
+| 00001
+| Successful completion
+| Omitted result
+| Successful completion with no return columns (e.g., update with no `RETURN` clause)
+
+| 02000
+| No data
+| 
+| Successful completion with an empty result 
+
+| 01000
+| Warning
+| 
+|
+
+| 03000
+| Informational
+| 
+| 
+
+|===
+
+In summary:
+
+* _successful completion_ (00000) is raised to indicate successful completion with a regular non-empty result.
+* _no data_ (02000) is raised to indicate successful completion with an empty result.
+* _successful completion — omitted result_ (00001) is raised to indicate successful completion with an omitted result.
+
+In the documentation, we call them `SUCCESS`, `OMITTED RESULT`, and `NO DATA`
+
+All query executions always return one of `SUCCESS`, `OMITTED RESULT`, or `NO DATA` results.
+
+link:https://neo4j.com/docs/cypher-manual/current/planning-and-tuning/[`EXPLAIN` queries] always give `OMITTED RESULT`.
+
+Classification, severity, position, and status parameters are not meaningful for these GQL statuses, so they are not included in the diagnostic record but the corresponding getter methods on server and driver side will be poly-filled with the default values.
```

Comment:
Drivers treat those values as absence and each driver follow a slightly different. Some made those values optional, others use a `UNKNOWN` classification/category and empty position, which is more appropriated for the language and driver api.